contribs/gmf/apps/mobile/js/mobile.js

```diff
@@ -24,6 +24,9 @@ goog.require('ol.layer.Tile');
 goog.require('ol.source.OSM');
 
 
+gmfModule.value('isMobile', true);
```

Comment:
Is the value constant? In this case you may call `gmfModule.constant()` instead.